ocaml/libs/tgroup/tgroup.ml

```diff
@@ -119,4 +134,37 @@ module Cgroup = struct
     Group.all
     |> List.map dir_of
     |> List.iter (fun dir -> Xapi_stdext_unix.Unixext.mkdir_rec dir 0o755)
+
+  let write_line_to_file filename s =
```

Comment:
Could you use any of the uniext function to implement this? https://github.com/xapi-project/xen-api/blob/master/ocaml/libs/xapi-stdext/lib/xapi-stdext-unix/unixext.mli#L83

Reply:
I was thinking of adding it in the unixext. Or a version of it. It's basically pidfile_write but the pid (in this case I want to pass the tid as a parameter) is passed as a parameter. I will also need something similar to write the cpu shares for cgroups.